File: lab2/lib/domain/Expense.dart
import 'Category.dart';
import 'currency.dart';

class Expense {
  String name;
  Category category;
  double amount;
  Currency currency;
  DateTime date;
  String description;

  Expense(this.name, this.category, this.amount, this.currency, this.date,
      this.description);
}

File: ManageExpenses/app/src/main/java/com/example/manageexpenses/Expense.kt
package com.example.manageexpenses

import java.io.Serializable
import java.time.LocalDate

data class Expense (val name : String,
                    val category : Category = Category.OTHERS,
                    val amount: Int = 0,
                    val currency : Currency = Currency.EUR,
                    val description : String,
                    val date : LocalDate = LocalDate.now()) : Serializable